src/components/Post/index.css

```diff
@@ -1,3 +1,7 @@
+.p-0 {
```

Comment:
give this class a more defining name to what it does, like `no-padding` or `padding-0`